mappings/net/minecraft/world/biome/DefaultBiomeCreator.mapping

```diff
@@ -90,3 +116,11 @@ CLASS net/minecraft/class_5478 net/minecraft/world/biome/DefaultBiomeCreator
 	METHOD method_30731 createWarpedForest ()Lnet/minecraft/class_1959;
 	METHOD method_30932 getSkyColor (F)I
 		ARG 0 temperature
+	METHOD method_31065 composeEndSpawnSettings (Lnet/minecraft/class_5485$class_5495;)Lnet/minecraft/class_1959;
```

Comment:
```suggestion
	METHOD method_31065 createEndSpawnSettings (Lnet/minecraft/class_5485$class_5495;)Lnet/minecraft/class_1959;
```

Reply:
`create` could be misleading because what it returns is biome. It indeed creates spawn settings in the method, but it also consumes spawn settings and generation settings to create biome.